index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>Joshua Ivie | Portfolio</title>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./assets/css/style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="shortcut icon" href="./assets/img/favicon.png">
+  <link rel="icon" href="./assets/img/favicon-32x32.png">
+  <link rel="apple-touch-icon" href="./assets/img/apple-icon.png">
+  <link rel="apple-touch-icon" sizes="72x72" href="./assets/img/apple-icon-72x72.png">
+  <link rel="apple-touch-icon" sizes="114x114" href="./assets/img/apple-icon-114x114.png">
+</head>
+
+<body>
+  <header>
+    <nav class="top-navigation">
+      <a href="index.html" class="logo">
+        <img src="./assets/img/logo.svg" alt="Joshua Ivie">
+      </a>
+      <div id="menuToggle" class="menu-toggle">
+        <span></span>
+        <span></span>
+        <span></span>
+      </div>
+    </nav>
+    <div class="hero">

```

Comment:
- [ ] The container of the section is a section tag. Instead of `div`, use `section`. This is to ensure best semantic practices.